common/src/main/java/io/homeassistant/companion/android/database/widget/MediaPlayerControlsWidgetEntity.kt

```diff
@@ -15,5 +15,7 @@ data class MediaPlayerControlsWidgetEntity(
     @ColumnInfo(name = "showSkip")
     val showSkip: Boolean,
     @ColumnInfo(name = "showSeek")
-    val showSeek: Boolean
+    val showSeek: Boolean,
+    @ColumnInfo(name = "showVolume")
+    val showVolume: Boolean
```

Comment:
As we are adding a new column in the database we also need to make sure to alter the table in `AppDatabase` and making sure to bump the database version appropriately.  You can see some examples in the file below on whats needed. 


https://github.com/home-assistant/android/blob/f800c85b522a0474a07544b1f031850a61f7d0f4/common/src/main/java/io/homeassistant/companion/android/database/AppDatabase.kt


When we make database changes its important to perform a test as a new user which you did here but to also test as an upgrade. To do that you want to first checkout the `master` branch and run that build first and login and create a media player widget. Then checkout your branch and run the build again to perform an update.

Reply:
I believe I fixed this correctly - adding a migration function and increasing the schema version. 

I tested by uninstalling, checking out 70e6207a049268db84e93f1a6eb60ae1e8d4fd62, installing on my device, signing into homeassistant, adding the media player widget, checking out my revision, running the build to install it, the migration seems to work, and the widget doesn't break.

Note: I set showVolume to '0' by default so that an update doesn't change an existing widget's layout.

I ran `./gradlew ktlintCheck`:

```

BUILD SUCCESSFUL in 2s
24 actionable tasks: 20 executed, 4 up-to-date
```

Reply:
awesome, once you resolve the conflict we can run the actions again and proceed :)

Reply:
Looks like after the most recent force-push the `AppDatabase` changes are missing

Reply:
terribly sorry, I corrected this on my end and force-pushed once again. My git skills are rusty.

So with c62720b83d5da4d756ecbbb58acb9073fae9a4df I have tested:

- install 1689a5b3ff1d7ceb93f52d67486fd62ec37508b7 and clear storage, sign in, create media player widget, install c62720b83d5da4d756ecbbb58acb9073fae9a4df, test existing widget, configure it to add volume buttons, test buttons.
- install c62720b83d5da4d756ecbbb58acb9073fae9a4df and clear storage, sign in, create media player widget, test volume buttons.
- ktlintCheck
- buttons are correct colours on light and dark themes